### ISSUE TITLE
Route endpoint finds some route but table endpoint's distance/duration is null

### ISSUE THREAD

**Issue by amirmasud** [Project-OSRM/osrm-backend#5266]
It's a bit weird. 
When we call 
`/table/v1/driving/51.41444495082490107,35.78089624643325806;51.41494141049,35.780812357317?destinations=1`
There is no route found. 

But when we call exactly the same coordinates in `route` endpoint, it finds.
`/route/v1/driving/51.41444495082490107,35.78089624643325806;51.41494141049,35.780812357317`

We tested it on both `latest` and `5.18` docker tags with MLD and all of them had `null` value on table endpoint :( 

**UPDATE**: 
we rerun with `CH` the algorithm again and there was **no null** on CH algorithm.

**Comment by amirmasud** [comment]
Also it's good to be mentioned that these two points are close to each other on a one-way road
Furthermore, when we swap the source and destination on `table` request it finds a route.

**Comment by amirmasud** [comment]
@jcoupey  I added an update for the question part. 
Also, on 42000 table request (points with less than 1 KM euclidean distance) we had about 2000 request with one or more nulls on MLD. 

we're going to test all those requests with CH too to test if it hasn't any nulls.

**Comment by amirmasud** [comment]
**UPDATE**
There were no `null` response in all of those 42000 table request on `CH` algorithm.

**Comment by jamiewinder** [comment]
I've just encountered this myself with:

`/table/v1/car/-1.2498919177722032,52.8002137485707;-1.9755007419310395,53.48975507479622?radiuses=1000;1000&generate_hints=false&sources=0&destinations=1&annotations=duration,distance`

The result is:
```
{"code":"Ok","sources":[{"distance":246.49217,"name":"Pasture Lane","location":[-1.249901,52.797999]}],"destinations":[{"distance":503.717194,"name":"","location":[-1.983072,53.49004]}],"durations":[[null]],"distances":[[null]]}
```
What options do I have to help with this? Is there any more detailed diagnostic information I can get at?

**Comment by mjjbell** [comment]
#5828 fixes this issue in v5.23.0
